test/amoc_api_scenarios_handler_SUITE.erl

```diff
@@ -71,6 +71,14 @@ put_scenarios_returns_400_and_error_when_scenario_is_not_valid(_Config) ->
     ?assertEqual(400, CodeHttp),
     ?assertEqual(#{<<"error">> => <<"invalid module">>}, Body).
 
+-define(PARSE_ERROR,
+        <<"\n                      [{2,erl_parse,[\"syntax error before: \",[]]}]}]\n">>).
+-if(?OTP_RELEASE >= 24).
+-undef(PARSE_ERROR).
+-define(PARSE_ERROR,
+        <<"\n                      [{{2,1},erl_parse,[\"syntax error before: \",[]]}]}]\n">>).
+-endif.
+
```

Comment:
```suggestion
-if(?OTP_RELEASE >= 24).
-define(PARSE_ERROR,
        <<"\n                      [{{2,1},erl_parse,[\"syntax error before: \",[]]}]}]\n">>).
-else.
-define(PARSE_ERROR,
        <<"\n                      [{2,erl_parse,[\"syntax error before: \",[]]}]}]\n">>).
-endif.
```

Reply:
I had something like that but Denys asked for it to be as it is now: https://github.com/esl/amoc/pull/136#discussion_r642712062

Reply:
Any motivation @DenysGonchar? It's less readable.

Reply:
Being honest, I don't remember, and now it's hard to tell what triggered me w/o original commit. Your version is fine and indeed it's more readable.